examples/vue/vue-storefront-2/scripts/add-resolutions.sh

```diff
@@ -2,4 +2,4 @@
 # You'll need to install `jq`: https://stedolan.github.io/jq/
 
 # Sets `@builder.io/sdk-vue` resolution symlinks in `package.json`
-echo "$(jq '.resolutions."@builder.io/sdk-vue" = "link:../../packages/sdks/output/vue"' package.json)" >package.json
+echo "$(jq '.resolutions."@builder.io/sdk-vue" = "link:../../../packages/sdks/output/vue"' package.json)" >package.json
```

Comment:
```suggestion
echo "$(jq '.resolutions."@builder.io/sdk-vue" = "link:../../../packages/sdks/output/vue"' package.json)" > package.json
```

Reply:
@mandx for some reason my VSCode shell formatter keeps removing that space...I don't know why 😄